src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -181,3 +183,8 @@ interface TriggeredInfo {
     val user: User?
     val build: Build?
 }
+
+interface BuildDependencyTrigger {
+    val dependsOnBuildConfiguration: BuildConfigurationId
```

Comment:
I'm not sure that 'dependsOn' is a good word here. How trigger can depend on something?